packages/core-plugin-api/api-report.md

```diff
@@ -29,6 +29,7 @@ export const alertApiRef: ApiRef<AlertApi>;
 export type AlertMessage = {
   message: string;
   severity?: 'success' | 'info' | 'warning' | 'error';
+  transient?: boolean;
```

Comment:
Discussed this a bit with @freben and thinking that we go forward with adding some form of option for this. Boolean is usually not the right choice for something like this though as it's not very forwards-compatible. What do you think about something like `display?: 'permanent' | 'transient'`? Happy to discuss naming alternatives though, not entirely happy with that one x)

Reply:
Thanks @Rugvip that's great news, sorry for the slow follow up. It might take me a longer than expected to make these changes as I'm going to be traveling with work and then taking a bit of vacation while I'm there

Reply:
OK, going to work on this later this week so let's talk about naming now. Honestly I have no strong opinion I'm happy to just use what you suggested - `display?: 'permanent' | 'transient'`. Are you still feeling unhappy with that now that some time has passed?

Reply:
Ok, I've implemented this change 👍